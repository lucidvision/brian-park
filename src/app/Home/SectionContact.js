import React from 'react'
import styled from 'styled-components'
import { BarSocialMedia } from 'components'

const StyledSection = styled.section`
  display: flex;
  justify-content: center;
`

export default function SectionContact (props) {
  return (
    <StyledSection>
      <BarSocialMedia />
    </StyledSection>
  )
}
