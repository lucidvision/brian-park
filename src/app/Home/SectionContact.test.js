import React from 'react'
import { shallow } from 'enzyme'
import SectionContact from './SectionContact'

describe('SectionContact', () => {
  const sectionContact = shallow(<SectionContact />)

  it('renders correctly', () => {
    expect(sectionContact).toMatchSnapshot()
  })

  it('renders a styled social media bar', () => {
    expect(sectionContact.find('Styled(BarSocialMedia)').exists()).toBeTruthy()
  })
})
