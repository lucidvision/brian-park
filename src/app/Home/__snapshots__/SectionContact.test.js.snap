// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionContact renders correctly 1`] = `
<styled.section>
  <Styled(BarSocialMedia)
    className="wow slideInUp"
  />
</styled.section>
`;
